src/mon/MonCommands.h

```diff
@@ -410,6 +414,13 @@ COMMAND("fs feature ls",
         "list available cephfs features to be set/unset",
 	"mds", "r")
 
+COMMAND("fs compat "
+        "name=fs_name,type=CephString "
+        "name=subop,type=CephChoices,strings=rm_compat|rm_incompat|add_compat|add_incompat "
```

Comment:
I guess some version of compat modification existed before so maybe I'm just behind the times, but what's the use case for these commands?

Reply:
Right now, just testing. I've been considering if I should flag them `HIDDEN`.